source/sync/configure/sync-settings.txt

```diff
@@ -25,23 +23,57 @@ Device Sync <enable-sync>`.
 Available Settings
 ------------------
 
+.. _sync-config-data-source:
+
+Cluster to Sync
+~~~~~~~~~~~~~~~
+
+The name of the linked Atlas cluster :ref:`data source <data-sources>` where
+you want to store your synced data.
+
+You cannot modify this field while Device Sync is enabled. You must
+:ref:`terminate Sync <pause-or-terminate-sync>` before you can select a
+different cluster.
+
+.. include:: /includes/note-flexible-sync-prerequisite.rst
+
 .. _flexible-sync:
 
 Sync Type
 ~~~~~~~~~
 
-Atlas Device Sync has two sync modes: Flexible Sync and the older
-Partition-Based Sync. Partition-Based Sync has 
-been deprecated and is disallowed for new Sync configurations. If you have 
-an existing app that uses Partition-Based Sync, you can migrate 
-to Flexible Sync. For more information, refer to :ref:`realm-sync-migrate-modes`.
+.. important:: Sync Type Displays Only for Partition-Based Sync Apps
+
+   The ability to select your app's Sync Type is only available for
+   organizations with at least one existing :ref:`Partition-Based Sync
+   <partition-based-sync>` app in their project.
+
+   Partition-Based Sync has been deprecated and is disallowed for new Sync
+   configurations. Instead, all new Sync configurations automatically default to the recommended Flexible Sync mode.
```

Comment:
Hmm. The setting is called Sync Type, but we usually use "sync mode" to talk about these two methods. It might be good to update our docs to use Sync Type. Could avoid conflating things with Development Mode. 😆 

For the purposes of this PR though, we could limit updating the wording to only what you've touched here.

Reply:
That's a good point. I wonder how often we're using "Sync Mode", but I am totally on board with your suggestion